libs/zzz/assets/src/executors/gen-assets/executor.ts

```
@@ -0,0 +1,80 @@
+import { generateIndexFromObj } from '@genshin-optimizer/common/pipeline'
+import { crawlObject, layeredAssignment } from '@genshin-optimizer/common/util'
+import { AssetData } from '@genshin-optimizer/zzz/assets-data'
+import { DM2D_PATH } from '@genshin-optimizer/zzz/dm'
+import { workspaceRoot } from '@nx/devkit'
+import * as fs from 'fs'
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)